boxoffice/models/order.py

```diff
@@ -89,3 +89,14 @@ def get_amounts(self):
             discounted_amount += line_item.discounted_amount
             final_amount += line_item.final_amount
         return order_amounts_ntuple(base_amount, discounted_amount, final_amount)
+
+    @property
+    def is_confirmed(self):
+        return self.status in [ORDER_STATUS.SALES_ORDER, ORDER_STATUS.INVOICE]
```

Comment:
Right now, we are only use sales order. Invoice, as a status, was added to account for orders for which tax invoices have been raised.